apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -264,6 +290,17 @@ export default connect(
             return <div key={i}>&gt; {rowValue.input}</div>;
           } else if (this.isValidOutput(rowValue)) {
             if (rowValue.fromConsoleLog) {
+              if (
+                experiments.isEnabled('mercator') &&
+                this.displayMap(rowValue.output)
+              ) {
+                return (
+                  <MercatorMap
```

Comment:
Keys for react objects in a map function should be unique strings and it's reasonable to assume that multiple map options might have the same latitude. Perhaps replacing this index `i` from line 285 will ensure that each key is unique?
(Documentation for keys, if interested: https://reactjs.org/docs/lists-and-keys.html#keys)

Reply:
Refactored as suggested!